src/navigation/pathnavigator_lua.inl

```diff
@@ -24,32 +24,45 @@
 
 namespace {
 
-// Returns true if a camera path is currently running, and false otherwise.
+/**
+ * Returns true if a camera path is currently running, and false otherwise.
+ *
+ * \return Whether a camera path is currently active, or not
+ */
 [[codegen::luawrap]] bool isFlying() {
     using namespace openspace;
     return global::openSpaceEngine->currentMode() == OpenSpaceEngine::Mode::CameraPath;
 }
 
-// Continue playing a paused camera path.
+/**
+ * Continue playing a paused camera path.
+ */
 [[codegen::luawrap]] void continuePath() {
     openspace::global::navigationHandler->pathNavigator().continuePath();
 }
 
-// Pause a playing camera path.
+/**
+ * Pause a playing camera path.
+ */
 [[codegen::luawrap]] void pausePath() {
     openspace::global::navigationHandler->pathNavigator().pausePath();
 }
 
-// Stops a path, if one is being played.
+/**
+ * Stops a path, if one is being played.
+ */
 [[codegen::luawrap]] void stopPath() {
     openspace::global::navigationHandler->pathNavigator().abortPath();
 }
 
-// Immediately skips to the end of the current camera path, if one is being played.
+/**
+ * Immediately skips to the end of the current camera path, if one is being played.
+ */
 [[codegen::luawrap]] void skipToEnd() {
     openspace::global::navigationHandler->pathNavigator().skipToEnd();
 }
 
+// @TODO: Split this up into two functions and then document (to remove the confusion)
```

Comment:
Discuss.  Let me know what you think here

Reply:
What would the two functions be in this case?

Reply:
Good question. I had some idea when I wrote this... I think it was to create another function that uses the up-direction, to get signatures like:

`openspace.pathnavigation.flyTo(identifier, std::optional<double> duration)`
`openspace.pathnavigation.flyToNodeAndPreserveUp(identifier, std::optional<double> duration)` (naming here is hard...)

Do they have to have different names though?  Or could we create "overloaded functions"?  Either way, writing the documentation for this seems almost impossible to do in an understandable way